### PR TITLE
Add German Translation

### PR DESCRIPTION
**Pull request by Sencaid**

### CODE REVIEW

File: common/resources/src/main/res/values/translations.xml
Comment (dead8309) [inline common/resources/src/main/res/values/translations.xml:1]: Should be `locale_de`

Comment (Sencaid) [reply common/resources/src/main/res/values/translations.xml:1]: Oh i forgot that, now its done

**Comment by Sencaid** [comment]
-___-

**Comment by Sencaid** [comment]
fixed it @dead8309

**Pull request merged**